daemon/network.go

```diff
@@ -235,6 +214,32 @@ func (daemon *Daemon) releaseIngress(id string) {
 	}
 }
 
+//AddLBAttachments sets the mapping of LB IP address per network
+func (daemon *Daemon) AddLBAttachments(lbAttachments map[string]string) error {
+
```

Comment:
Can you remove this empty line here (and other functions you added) to make the coding style match the existing code?

Reply:
Done.